src/test/mochitest/browser_dbg-search-file.js

```diff
@@ -35,11 +35,12 @@ add_task(async function() {
   const el = getFocusedEl(dbg);
 
   type(dbg, "con");
-  await waitForSearchState(dbg);
 
   const state = cm.state.search;
 
   pressKey(dbg, "Enter");
+  pressKey(dbg, "Enter");
```

Comment:
I don't know why the second ent here seems to be the trick.

Reply:
looks like it might be a timing issue